### PR TITLE
chore(copy): make on_error default to abort

### PR DESCRIPTION
**Pull request by ariesdevil**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Change `ON_ERROR` default value to `abort`.

Closes #issue

### CODE REVIEW

File: scripts/ci/ci-run-sqllogic-tests.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@ fi
 echo "Run suites using argument: $RUN_DIR"
 
 echo "Starting databend-sqllogic tests"
-target/${BUILD_PROFILE}/databend-sqllogictests --handlers ${TEST_HANDLERS} ${RUN_DIR} --enable_sandbox --parallel 8 --debug
+target/${BUILD_PROFILE}/databend-sqllogictests --handlers ${TEST_HANDLERS} ${RUN_DIR} --skip_dir management --enable_sandbox --parallel 8 --debug
```
Comment (BohuTANG) [inline scripts/ci/ci-run-sqllogic-tests.sh:20]: Why skip management test?

Comment (ariesdevil) [reply scripts/ci/ci-run-sqllogic-tests.sh:20]: Tests in management dir need the server run on management mode, it does not affect the ci run cause our ci has a separate action(test_sqllogic_management_mode_linux) to run this test.
It just affect do `make sqllogic-test` on local machine.

**Comment by ariesdevil** [comment]
The failed sqllogic tests show:
```
[SQL] explain select * from t, t1, t2 where t.a = t1.a and t1.a = t2.a and t2.a = t.a
[Diff] (-expected|+actual)
    HashJoin
    ├── join type: INNER
    ├── build keys: [t.a (#0), t.a (#0)]
    ├── probe keys: [t2.a (#2), t1.a (#1)]
    ├── filters: []
    ├── estimated rows: 0.01
    ├── TableScan(Build)
-   │   ├── table: default.join_reorder.t
+   │   ├── table: default.default.t
    │   ├── read rows: 1
    │   ├── read bytes: 39
    │   ├── partitions total: 1
    │   ├── partitions scanned: 1
    │   ├── pruning stats: [segments: <range pruning: 1 to 1>, blocks: <range pruning: 1 to 1, bloom pruning: 0 to 0>]
    │   ├── push downs: [filters: [], limit: NONE]
    │   └── estimated rows: 1.00
    └── HashJoin(Probe)
        ├── join type: INNER
        ├── build keys: [t1.a (#1)]
        ├── probe keys: [t2.a (#2)]
        ├── filters: []
        ├── estimated rows: 0.01
        ├── TableScan(Build)
-       │   ├── table: default.join_reorder.t1
+       │   ├── table: default.default.t1
        │   ├── read rows: 10
        │   ├── read bytes: 65
        │   ├── partitions total: 1
        │   ├── partitions scanned: 1
        │   ├── pruning stats: [segments: <range pruning: 1 to 1>, blocks: <range pruning: 1 to 1, bloom pruning: 0 to 0>]
        │   ├── push downs: [filters: [], limit: NONE]
        │   └── estimated rows: 10.00
        └── TableScan(Probe)
-           ├── table: default.join_reorder.t2
+           ├── table: default.default.t2
            ├── read rows: 100
            ├── read bytes: 172
            ├── partitions total: 1
Error: SelfError("sqllogictest failed")
            ├── partitions scanned: 1
            ├── pruning stats: [segments: <range pruning: 1 to 1>, blocks: <range pruning: 1 to 1, bloom pruning: 0 to 0>]
            ├── push downs: [filters: [], limit: NONE]
            └── estimated rows: 100.00
at tests/sqllogictests/suites/mode/standalone/explain/join_reorder/cycles.test:16
```
Have we changed anything recently? cc @xudong963

**Comment by ariesdevil** [comment]
And these on my local machine
```
Test finished, Total 2 records failed to run
0: query result mismatch:
[SQL] explain select * from numbers(1) t, numbers(2) t1 where t.number = t1.number
[Diff] (-expected|+actual)
-   Exchange
-   ├── exchange type: Merge
-   └── HashJoin
-       ├── join type: INNER
-       ├── build keys: [t.number (#0)]
-       ├── probe keys: [t1.number (#1)]
-       ├── filters: []
-       ├── estimated rows: 2.00
-       ├── Exchange(Build)
-       │   ├── exchange type: Hash(t.number (#0))
-       │   └── TableScan
-       │       ├── table: default.system.numbers
-       │       ├── read rows: 1
-       │       ├── read bytes: 8
-       │       ├── partitions total: 1
-       │       ├── partitions scanned: 1
-       │       ├── push downs: [filters: [], limit: NONE]
-       │       └── estimated rows: 1.00
-       └── Exchange(Probe)
-           ├── exchange type: Hash(t1.number (#1))
-           └── TableScan
-               ├── table: default.system.numbers
-               ├── read rows: 2
-               ├── read bytes: 16
-               ├── partitions total: 1
-               ├── partitions scanned: 1
-               ├── push downs: [filters: [], limit: NONE]
-               └── estimated rows: 2.00
+   HashJoin
+   ├── join type: INNER
+   ├── build keys: [t.number (#0)]
+   ├── probe keys: [t1.number (#1)]
+   ├── filters: []
+   ├── estimated rows: 2.00
+   ├── TableScan(Build)
+   │   ├── table: default.system.numbers
+   │   ├── read rows: 1
+   │   ├── read bytes: 8
+   │   ├── partitions total: 1
+   │   ├── partitions scanned: 1
+   │   ├── push downs: [filters: [], limit: NONE]
+   │   └── estimated rows: 1.00
+   └── TableScan(Probe)
+       ├── table: default.system.numbers
+       ├── read rows: 2
+       ├── read bytes: 16
+       ├── partitions total: 1
+       ├── partitions scanned: 1
+       ├── push downs: [filters: [], limit: NONE]
+       └── estimated rows: 2.00
at tests/sqllogictests/suites/mode/cluster/exchange.test:1

1: query result mismatch:
[SQL] explain select t1.a from t1 where a > 0;
[Diff] (-expected|+actual)
-   Exchange
-   ├── exchange type: Merge
-   └── Filter
-       ├── filters: [gt(t1.a (#0), to_int32(0_u8))]
-       ├── estimated rows: 0.00
-       └── TableScan
-           ├── table: default.default.t1
-           ├── read rows: 0
-           ├── read bytes: 0
-           ├── partitions total: 0
-           ├── partitions scanned: 0
-           ├── push downs: [filters: [gt(t1.a (#0), 0_i32)], limit: NONE]
-           ├── output columns: [0]
-           └── estimated rows: 0.00
+   Filter
+   ├── filters: [gt(t1.a (#0), to_int32(0_u8))]
+   ├── estimated rows: 0.00
+   └── TableScan
+       ├── table: default.default.t1
+       ├── read rows: 0
+       ├── read bytes: 0
+       ├── partitions total: 0
+       ├── partitions scanned: 0
+       ├── push downs: [filters: [gt(t1.a (#0), 0_i32)], limit: NONE]
+       ├── output columns: [0]
+       └── estimated rows: 0.00
at tests/sqllogictests/suites/mode/cluster/04_0002_explain_v2.test:13
```

**Comment by BohuTANG** [comment]
https://github.com/datafuselabs/databend/issues/9734

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**